pkg/execute/executor.go

```diff
@@ -146,6 +149,14 @@ const (
 	FilterDisable FiltersAction = "disable"
 )
 
+// infoAction for options in Info commands
+type infoAction string
+
+// Info command options
+const (
+	InfoList infoAction = "list"
```

Comment:
Avoid exporting variables if not required

Reply:
done

Reply:
Done